packages/babel-plugin-transform-wpcalypso-async/package.json

```diff
@@ -1,5 +1,8 @@
 {
 	"name": "@automattic/babel-plugin-transform-wpcalypso-async",
 	"version": "1.0.1",
-	"private": true
+	"private": true,
+	"devDependencies": {
+		"@babel/plugin-syntax-jsx": "7.2.0"
```

Comment:
Sorry I'm late here, but I think we need to be avoiding `devDependencies` in packaged.

If this belongs in a `devDependency` —sounds like it does— it should be at the root or we can't expect it to be installed.

Reply:
Yeah, this has been puzzling me, too. See https://github.com/Automattic/wp-calypso/pull/33126#discussion_r285131552 for more detail -- it _might_ be okay here...

Reply:
Resolved per #33161 and a rebase :grimacing: